Manifold/TermType+Construction.swift

```diff
@@ -152,7 +152,7 @@ extension TermType {
 		return cata {
 			$0.analysis(
 				ifApplication: max,
-				ifLambda: { max($0.0, $0.1) },
+				ifLambda: { $0 < 0 ? max($1, $2) : max($0, $1) },
```

Comment:
When we construct a lambda whose parameter is unused, we assign it the variable `-1`. In this case, we want to ignore the variable it binds, and consider variables bound in its body.